rose-suite.conf

```diff
@@ -0,0 +1,46 @@
+[template variables]
```

Comment:
Similar comment to the `opt/TEMPLATE` comment above. `configure_script_yaml.py` would be creating this file. So should this template be here anymore?